app/Server.hs

```diff
@@ -25,6 +25,8 @@ runServer = do
     staticDir <- getStaticDir
     aboutContents <- LazyIO.readFile $ markdownPath ++ "README.md"
     privacyContents <- LazyIO.readFile $ markdownPath ++ "PRIVACY.md"
+    -- Create db folder if it doesn't exist
+    createDirectoryIfMissing False "db"
```

Comment:
This path shouldn't be hard-coded to "db" - it's set in the configuration file, I believe.